aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -3211,17 +3211,17 @@ static void linalg_lstsq_out_info(
   }
 }
 
-static std::string get_default_lstsq_driver(c10::optional<std::string> driver, const Tensor& input) {
+static std::string get_default_lstsq_driver(c10::optional<c10::string_view> driver, const Tensor& input) {
   // if `driver` is empty, we set driver_str to "gels" if working with CUDA tensors,
   // otherwise to "gelsy" driver.
   std::string driver_str;
   // check whether the user provided name is a valid driver name
   if (driver.has_value()) {
-    driver_str = driver.value();
+    driver_str = std::string(driver.value().data());
```

Comment:
Note to self: we don't need these `.data()` calls since `std::string()` knows how to handle a `c10::string_view`